rita_exit/src/database/mod.rs

```diff
@@ -381,8 +380,35 @@ pub fn setup_clients(
         &wg_clients,
         settings::get_rita_exit().exit_network.wg_tunnel_port,
         &settings::get_rita_exit().exit_network.wg_private_key_path,
+        "wg_exit",
+    );
+
+    // Setup new tunnels
+    let exit_status_new = KI.set_exit_wg_config(
+        &wg_clients,
+        settings::get_rita_exit().exit_network.wg_new_tunnel_port,
+        &settings::get_rita_exit().network.wg_private_key_path,
+        "wg_exit_new",
     );
 
+    // Setup or verify that an ipv6 route exists for each client
+    let new_wg_exit_clients: HashMap<WgKey, SystemTime> = KI
+        .get_last_handshake_time("wg_exit_new")
+        .expect("There should be a new wg_exit interface")
+        .into_iter()
+        .collect();
+    let wg_exit_clients: HashMap<WgKey, SystemTime> = KI
+        .get_last_handshake_time("wg_exit")
+        .expect("There should be a wg_exit interface")
+        .into_iter()
+        .collect();
+
+    for c in clients_list.iter() {
+        let interface =
+            get_client_interface(c, new_wg_exit_clients.clone(), wg_exit_clients.clone())?;
+        KI.setup_client_routes(c.internet_ipv6.clone(), c.mesh_ip.clone(), &interface)
```

Comment:
This is going to have to be re-thought for a number of issues

1) we only go and run this code when we know there's a change to enforcement status or registration list (or maybe I'm not quite right and this is above that level double check me)
2) lots of clients remain disconnected and don't have an active handshake on either tunnel. Now this isn't the end of the world since we don't need a route for a client that's offline. But we'll need to be able to detect them coming online and attach them to the right interface. Including situations where clients upgrade and switch interfaces while the exit is online. Maybe instead write a function that's more imdepotent and just checks that every route goes to the most recent handshaked interface.

Reply:
I moved this upward where it is called every tick. In the case when neither interface has a handshake, i simply skip setting up routes. I set them up when i finally see a handshake